tests/integration/features/zoom.feature

```diff
@@ -17,19 +17,23 @@ Feature: Zooming in and out
 
     Scenario: Zooming in with count
         When I run :zoom-in with count 2
-        Then the message "Zoom level: 120%" should be shown.
-        And the zoom should be 120%
+        Then the message "Zoom level: 125%" should be shown.
+        And the zoom should be 125%
```

Comment:
Seeing `zoom-levels` is set to something including `120%` but not `125%`, why did this change?